### DISCUSSION TITLE
Invoke wayland/lxqt from multi-user.target?

### DISCUSSION THREAD

**Discussion by raekuul** [lxqt/lxqt#2572] [Q&A]
How do I launch the wayland version of lxqt from the command line?  My main machine defaults to `multi-user.target` so I do need to learn this.

**Comment by stefonarch** [comment]
I'm using those scripts from tty (for testing purposes with test users):
https://github.com/stefonarch/LXQt-Wayland-files/blob/main/start_scripts/

Read also the README.md
